metriq_gym/qplatform/device.py

```diff
@@ -36,3 +373,50 @@ def _(device: BraketDevice) -> rx.PyGraph:
         rx.PyGraph,
         rx.networkx_converter(nx.Graph(device._device.topology_graph.to_undirected())),
     )
+
+
+@connectivity_graph.register
+def _(device: LocalDevice) -> rx.PyGraph:
+    """Get connectivity graph for local devices.
+    
+    Args:
+        device: LocalDevice instance
+        
+    Returns:
+        rx.PyGraph: Connectivity graph for the device
+    """
+    num_qubits = device.num_qubits
+    
+    # Limit to reasonable size for benchmarks
+    if num_qubits > 127:
+        num_qubits = 20  # Reasonable default for benchmarking
```

Comment:
We also silently update the number of qubits here which could cause some misleading results to occur.